### PR TITLE
Addition of CSS

### PR DESCRIPTION
**Pull request by CBreul**
Added styling to all pages - spacing is a little funky, as is my attempt at the hamburger mobile menu.

### CODE REVIEW

File: pages/npcs.html [hunk 1]
```diff
@@ -4,166 +4,200 @@
...
+                <h3>Elemental Locus of Battering Winds</h3>
+                <hr class="header2">
+                <h4>Air Elemental</h4>
             </header>
             <section>
-                <p>
+                <p class="chardesc">
                    An air elemental formed on the bones of a Primal Pteradon.
                 </p>
                <h6>Defeated.</h6>
```
Comment (axamei) [inline pages/npcs.html]: think of the headings and heading levels like a table of contents. i recommend that you don't use `h5` or `h6` on these profiles and instead use a `span` tag with a class to style it as desired.

**Review by axamei — CHANGES_REQUESTED** [review]
small change to your use of headings and heading levels.

File: pages/npcs.html (hunk 1 above)
Comment (CBreul) [reply pages/npcs.html]: @axamei Got it! I have just made the updates.

**Review by axamei — APPROVED** [review]
👍

**Pull request merged**